### DISCUSSION TITLE
cython version

### DISCUSSION THREAD

**Discussion by ff781** [networkx/networkx#7686] [General]
https://groups.google.com/g/networkx-discuss/c/SES2fPG0f9c?pli=1

was a cython version ever built?

**Comment by dschult** [comment]
Not that I know of. The latest speedup apporach has been developing over the last 3 years and is quite effective for the functions that are provided. We have a backend system where, once you install a backend like nx-cugraph, you can change an environment variable to have your networkx code run on the gpu. See https://networkx.org/documentation/stable/backends.html
